caffe2/CMakeLists.txt

```diff
@@ -1261,6 +1261,12 @@ if(BUILD_STATIC_RUNTIME_BENCHMARK)
   target_link_libraries(static_runtime_test torch_library gtest_main)
 endif()
 
+if(BUILD_TENSOREXPR_BENCHMARK)
```

Comment:
just a heads up that if I enable this and BUILD_STATIC_RUNTIME_BENCHMARK I get into a weird state where cmake complains

```
CMake Error at caffe2/CMakeLists.txt:1265 (add_subdirectory):
  The binary directory

    /mnt/ssd1/bwasti/pytorch/build/bin

  is already used to build a source directory.  It cannot be used to build                                                                                                                                                                      source directory                                                                                                                                                                                                                                                                                                                                                                                                                                                                              /mnt/ssd1/bwasti/pytorch/benchmarks/cpp/tensorexpr
```

Reply:
Thanks, fixed it!